test/docker-e2e/e2e_upgrade_test.go

```diff
@@ -0,0 +1,306 @@
+package docker_e2e
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+	"testing"
+
+	"celestiaorg/celestia-app/test/docker-e2e/dockerchain"
+
+	"github.com/celestiaorg/tastora/framework/testutil/wait"
+	tastoratypes "github.com/celestiaorg/tastora/framework/types"
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+)
+
+// TallyResponse represents the JSON response from the signal tally query
+// We had to define it again because the JSON response type is not a number but string
```

Comment:
[not blocking]

the entire rest of the repo and almost all other go repos by default wrap comments instead of adding new lines arbitrarily. not going to block on silly stuff but fixing would be nice

Reply:
the comments are wrapped now.